src/requirements.txt

```diff
@@ -26,3 +32,6 @@ setuptools==60.9.3
 simplegeneric==0.8.1
 sympy==1.9
 terminado==0.13.1
+# ensure pip installs all deps
+jupyter-console==6.0.0
+pytest==6.2.5
```

Comment:
using a requirements file (versus running `pip install ...`, `pip` doesn't do a very good job of ensuring all dependencies are in place before installing a module. Moving these two modules to the end of the file ensures that pip installs all necessary prerequisites.